### PR TITLE
Add configs for web-engine try jobs.

### PR DESCRIPTION
**Pull request by godofredoc**
Also ran yapf to reformat the file.

Context:
  https://github.com/flutter/flutter/issues/46782

### CODE REVIEW

**Comment by digiter** [comment]
I would split this into 2 PRs, one for the config, another for the yapf formatting,  otherwise it's hard to review.

**Comment by godofredoc** [comment]
Done created https://github.com/flutter/infra/pull/86 for yapf update I'll merge and update once it is submitted.

**Comment by godofredoc** [comment]
Updated the PR to pickup the yapf formatting changes. PTAL

File: config/main.star [hunk 1]
```diff
@@ -165,7 +166,6 @@ def console_view(name, repo, refs=['refs/heads/master'], exclude_ref=None):
     )
 
 
-console_view('cocoon', COCOON_GIT)
```
Comment (digiter) [inline config/main.star]: Intended to remove this console view?

Comment (godofredoc) [reply config/main.star]: Yes, this is the empty presubmit console for cocoon.

**Pull request merged**